core/server/api/configuration.js

```diff
@@ -20,7 +20,7 @@ function getAboutConfig() {
         version: ghostVersion.full,
         environment: config.get('env'),
         database: config.get('database').client,
-        mail: _.isObject(config.get('mail')) ? config.get('mail').transport : ''
+        mail: Object(config.get('mail')) === config.get('mail') ? config.get('mail').transport : ''
```

Comment:
Hmm this replacement is also true when config.get('mail') is an Array. But also `isObject` was detecting that.
I think what we need here is `isPlainObject`.